docs/Data Provider Nodes/sport-monks-oracle.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: "SportMonks Sports Data Oracle"
+permalink: "docs/sport-monks-oracle/"
+---
+
+SportMonks is a provider of reliable sport data feeds, powered by a fast, highly available, and easy to implement API. They currently provide live data on three sports: soccer, F1, and cricket. They offer a variety of data such as live scores, player statistics, team stats, seasons stats, bookmakers and markets, (live) odds, predictions and coverage across more than 1,200 leagues. This oracle will initially provide Match Results and Toss Results for the 2021 Indian Premier League Cricket season.
+
+# Steps For Using This Oracle
+
+- Write and deploy your [Chainlink](../intermediates-tutorial/) contract using the network details below
+- Fund it with [LINK](../link-token-contracts/)
+- Call your [request method](#section-chainlink-examples)
```

Comment:
Anchor broken, and not sure if this anchor will work after #54 is merged?

Reply:
doesn't but we'll find a way to get anchors back

Reply:
Can we at least get this one to work? I noticed that the adapters seem to work fine.